docs/terraform.md

```diff
@@ -18,8 +18,8 @@ For example, a complete setup might be:
 ```
 export KOPS_STATE_STORE=s3://<somes3bucket>
 export CLUSTER_NAME=<kubernetes.mydomain.com>
-${GOPATH}/bin/kops create cluster ${NAME} --zones us-east-1c
-${GOPATH}/bin/kops update cluster ${NAME} --target=terraform
```

Comment:
we have an inconsistency with the docs NAME vs CLUSTER_NAME ~ does this help it?